cmds/standings.js

```diff
@@ -0,0 +1,50 @@
+const ora = require('ora');
+const moment = require('moment');
+const request = require('request');
+const URLS = require('../constants');
+const config = require('../config');
+const helpers = require('../helpers');
+const leagueIds = require('../leagueIds');
+
+const updateMessage = helpers.updateMessage;
+const standingsHelper = helpers.standingsHelper;
+
+const footballRequest = request.defaults({
```

Comment:
Let's move this to separate utility file for `scores`, `standings` and `fixtures`.

Reply:
You mean creating separate files for scores, fixtures and standings instead of having one helper.js right?

Reply:
No, let's keep `request_utils.js` which exports this object.